wordpress-mobile/gutenberg-mobile
# Unsupported Block

### Description:

This PR implements the unsupported block for blocks we don't offer support for.

Fixes #50.

### Details:

This PR introduces a new block type to display all the blocks we don't support.  Adding support for a block happens through regular registration of the block type in Gutenberg.

Next steps: introduce this new block type as the native version of the `freeform` block.

### Known issues / limitations:

After the changes submitted by @koke in 
https://github.com/wordpress-mobile/gutenberg-mobile/pull/127, I could no longer retrieve the original block name to display in visual mode.

This should be fine though, as unsupported blocks could be changing once this is merged: https://github.com/WordPress/gutenberg/pull/8274

### Testing:

- Load the standard post content.
- Switch to HTML mode and make sure the block contents are properly maintained.
